test/run-k8s-integration-local.sh

```diff
@@ -73,6 +73,24 @@ make -C "${PKGDIR}" test-k8s-integration
 # --gce-zone="us-central1-c" --num-nodes=${NUM_NODES:-3} --gke-release-channel="rapid" --deployment-strategy="gke" \
 # --use-gke-managed-driver=true --teardown-cluster=true
 
+# This version of the command brings up (and subsequently tears down) a GKE
+# cluster using a release channel, and a given worker node version and with managed GCE PersistentDisk CSI driver add-on enabled, and points to
+# the local K8s repository to get the e2e test binary.
+# ${PKGDIR}/bin/k8s-integration-test --run-in-prow=false --service-account-file=${GCE_PD_SA_DIR}/cloud-sa.json \
+# --test-focus="External.Storage.*snapshot" --local-k8s-dir=$KTOP --storageclass-files=sc-standard.yaml,sc-balanced.yaml,sc-ssd.yaml \
+# --snapshotclass-file=pd-volumesnapshotclass.yaml --do-driver-build=false --teardown-driver=false \
+# --gce-zone="us-central1-c" --num-nodes=${NUM_NODES:-3} --gke-release-channel="rapid" --deployment-strategy="gke" \
+# --use-gke-managed-driver=true --teardown-cluster=true --node-version="1.16"
+
+# This version of the command brings up (and subsequently tears down) a GKE
```

Comment:
I don't know if this command is necessary.

Really all these commented versions I think are to easily test common cases. But I find if there's too many it's hard to confirm all the flags are right so I fiddle with the flags anyway in addition to uncommenting. If there's two many different commands the file gets hard to navigate.

Reply:
What if we just have one or two commands, and we put each argument on a separate line and have it commented out if we don't want it by default?

Reply:
Removing this change. We can take up the cleanup in a different patch

Reply:
I like @msau42's suggestion.

It's not a huge deal either way as usually I have a locally tweaked version anyway. But I do want to make sure we don't clutter up this script too much.

+1 to removing the change for now and cleaning up after more bikeshedding....